src/types/strategies.ts

```diff
@@ -1,18 +1,57 @@
+import { PresentationChartBarIcon } from '@heroicons/react/24/outline';
 import { CurrencyExt } from '@interlay/interbtc-api';
-import Big from 'big.js';
+import { ForwardRefExoticComponent, RefAttributes } from 'react';
+
+import { WRAPPED_TOKEN } from '@/config/relay-chains';
 
 enum StrategyRisk {
   LOW = 'low',
   MEDIUM = 'medium',
   HIGH = 'high'
 }
 
-type StrategyData = {
-  interestType: 'apy' | 'apr';
-  interestPercentage: Big;
-  risk: StrategyRisk;
+enum StrategyType {
+  BTC_LOW_RISK = 'BTC_LOW_RISK'
```

Comment:
our enums could be the path to the strategy page.

`BTC_LOW_RISK` = "btc-low-risk"

Reply:
Okay, makes sense :)